source/server/admin/stats_handler.h

```diff
@@ -79,9 +79,10 @@ class StatsHandler : public HandlerContextBase {
    * and for rendering in /stats?format=html. We share the same UrlHandler in
    * both contexts by defining an API for it here.
    *
+   * @prams active_mode excludes 'usedonly' -- that setting is required for active mode.
```

Comment:
@prams?

And this description needs some work. Is it saying that if active_mode is true, 'usedonly' stats are not included? This might be a thing where someone with all the context would understand it, but as you know I've been touching stats, and I have no idea what "that setting is required for active mode" means, nor usedonly. A clearer explanation (maybe even a context link?) would be good.

Reply:
reworded; ptal.